src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml

```diff
@@ -46,13 +46,15 @@
                     Width="12"
                     Height="12"
                     Margin="0,0,4,0"
+                    Foreground="{x:Bind help:OptionalColorBrushCacheProvider.Convert(Foreground, x:True), Mode=OneWay}"
                     SourceKey="{x:Bind Icon, Mode=OneWay}"
                     SourceRequested="{x:Bind help:IconCacheProvider.SourceRequested}"
                     Visibility="{x:Bind HasIcon, Mode=OneWay}" />
 
                 <TextBlock
                     VerticalAlignment="Center"
                     FontSize="12"
+                    Foreground="{x:Bind help:OptionalColorBrushCacheProvider.Convert(Foreground, x:True), Mode=OneWay}"
```

Comment:
can we like, pass in `ActualTheme` through the binding to let the thing know which theme to be in?

Reply:
Yeah, we could do that. I think based on the other comment we just need to figure out what the patterns we want here are for the various scenarios. Then we can figure out how to adjust this to support it after. Let's follow-up on this as a follow-on to the initially enablement here.